go/teams/teams.go

```diff
@@ -1302,3 +1304,37 @@ func (t *Team) precheckLinkToPost(ctx context.Context, sigMultiItem libkb.SigMul
 	}
 	return precheckLinkToPost(ctx, t.G(), sigMultiItem, t.chain(), me.ToUserVersion())
 }
+
+// Try to run `post` (expected to post new team sigchain links).
+// Retry it several times if it fails due to being behind the latest team sigchain state.
+// Passes the attempt number (initially 0) to `post`.
+func RetryOnSigOldSeqnoError(ctx context.Context, g *libkb.GlobalContext, post func(ctx context.Context, attempt int) error) (err error) {
+	defer g.CTraceTimed(ctx, "RetryOnSigOldSeqnoError", func() error { return err })()
+	const nRetries = 3
+	for i := 0; i < nRetries; i++ {
+		g.Log.CDebugf(ctx, "| RetryOnSigOldSeqnoError(%v)", i)
+		err = post(ctx, i)
+		if isSigOldSeqnoError(err) {
+			// This error means retry
+			continue
+		}
+		return err
+	}
+	if err == nil {
+		// Should never happen
+		return fmt.Errorf("failed retryable team operation")
```

Comment:
Maybe make the error here reflect that some invariant got broken in the code here, instead of being generic?

Reply:
I was thinking I don't want users to see it. So a dev will see this and look in the code and see the comment?

Reply:
If it's actually never supposed to happen, then I'm ok with users seeing it in the event that it _does_ happen, I think.